core/common/src/main/java/alluxio/security/authentication/DefaultAuthenticationServer.java

```diff
@@ -80,7 +80,7 @@ public DefaultAuthenticationServer(String hostName, AlluxioConfiguration conf) {
         Executors.newScheduledThreadPool(1, ThreadFactoryUtils.build("auth-cleanup", true));
     mScheduler.scheduleAtFixedRate(this::cleanupStaleClients, mCleanupIntervalMs,
         mCleanupIntervalMs, TimeUnit.MILLISECONDS);
-    mImpersonationAuthenticator = new ImpersonationAuthenticator(conf);
+    mImpersonationAuthenticator = ImpersonationAuthenticator.getInstance();
```

Comment:
Why we limit `ImpersonationAuthenticator` to be singleton, this coulc limit extensible about authentication, e.g. support multiply authentication server within one alluxio cluster.

Reply:
fix.